dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1632,4 +1632,6 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
 
 sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
 
-query.or.post.is.null=Request query string or post data is null.
+query.or.post.is.null=CWWWC0009E: Request query string or post data is null.
+query.or.post.is.null.explanation=Application tries to parse a null request parameter or a post body.
```

Comment:
Change "Application tries to parse a null request parameter or a post body." to "The application tried to parse a null HTTP request parameter or a null HTTP POST request body."